spec/models/accounting/quickbooks/updater_spec.rb

```diff
@@ -35,5 +44,73 @@
         expect { subject.update }.not_to raise_error
       end
     end
+
+    context 'with new JournalEntry' do
+      before do
+        allow(subject).to receive(:changes).and_return('JournalEntry' => [journal_entry])
+      end
+
+      let(:last_updated_at) { 30.days.ago }
+      let(:qb_id) { 34 }
+      let(:journal_entry) { instance_double(Quickbooks::Model::JournalEntry, id: qb_id) }
+
+      subject { described_class.new(connection) }
+
+      it 'creates a new transaction with the proper qb_id' do
+        expect { subject.update }.to change { Accounting::Transaction.where(qb_id: qb_id).count }.by 1
+      end
+
+      it 'is created with proper type' do
+        subject.update
+        transaction = Accounting::Transaction.where(qb_id: qb_id).take
+
+        expect(transaction.qb_transaction_type).to eq 'JournalEntry'
+      end
+
+      it 'is created with quickbooks data' do
+        subject.update
+        transaction = Accounting::Transaction.where(qb_id: qb_id).take
+
+        expect(transaction.quickbooks_data).not_to be_empty
+      end
```

Comment:
Could you perhaps condense all the above into one? There is a fair amount of duplication here and I think reducing it would make these easier to read...